src/Bll/StatementBLL.php

```diff
@@ -397,7 +398,12 @@ public function acceptPartialFundsById(int $statementId, ?StatementDTO $statemen
                 );
             }
 
-            $this->rejectFundsById($statementId, StatementDTO::createEmpty()->setDescription('Reversal of partial acceptance for reserve ' . $statementId));
+            $this->rejectFundsById($statementId,
+                StatementDTO::createEmpty()
+                    ->setDescription('Refund for initial bet ' . $statementId)
```

Comment:
This is not acceptable. If you are not passing the DTO, you should choose a generic message.